framework/src/modules/interoperability/base_interoperability_endpoint.ts

```diff
@@ -148,6 +149,26 @@ export abstract class BaseInteroperabilityEndpoint<
 		};
 	}
 
+	public async getChainValidators(
+		context: ModuleEndpointContext,
+		chainID: Buffer,
+	): Promise<ChainValidators> {
+		const interoperabilityStore = this.getInteroperabilityStore(context);
+		const chainValidators = await interoperabilityStore.getChainValidators(chainID);
+
+		return chainValidators;
+	}
+
+	public async isChainIDAvailable(
```

Comment:
same here, use `ChainAccountStore` store directly instead of using `interoperabilityStore`